### DISCUSSION TITLE
More user roles

### DISCUSSION THREAD

**Discussion by CHUYKOPS** [outline/outline#4741] [Ideas]
Good afternoon!

Thanks for the great service, our team is happy to use the solution!

It would be great to be able to create more roles for users.

For example:

The administrator creates a new collection for the department and assigns a responsible person to add users to the collection.
In the control panel, this employee can view either all users or all users except administrators and add them to the collection.
This functionality will greatly simplify the life of administrators.

**Comment by tommoor** [comment]
Yes, there is an existing plan for a collection-admin type role 👍

**Reply by CHUYKOPS** [reply]
> Yes, there is an existing plan for a collection-admin type role 👍

Thanks, will wait for it!
